ERCS/erc-7540.md

```diff
@@ -46,13 +46,13 @@ The existing definitions from [ERC-4626](./eip-4626.md) apply. In addition, this
 
 ### Request Flows
 
-[ERC-7540 Vaults](./eip-7540.md) MUST implement one or both of asynchronous deposit and redemption Request flows. If either flow is not implemented in a Request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
+[ERC-7540 Vaults](./eip-7540.md) MUST implement one or both of asynchronous deposit and redemption Request flows. If both flows are not implemented in a Request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
```

Comment:
The previous phrasing is correct here: say if only the async deposit request flow is not implemented, then the synchronous (ERC4626) deposit flow should be used.